packages/vkui/src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -148,6 +158,14 @@ export interface SelectProps extends NativeSelectProps, FormFieldProps, TrackerO
   onClose?: VoidFunction;
   onOpen?: VoidFunction;
   icon?: React.ReactNode;
+  /**
+   * Кастомная кнопка для очистки значения
+   */
+  clearButton?: React.ReactNode;
+  /**
+   * Если `true`, то справа будет отображаться кнопка для очистки значения
+   */
+  allowClear?: boolean;
```

Comment:
```suggestion
  allowClearButton?: boolean;
```
просто проп для замены называется `clearButton` :)